XMonad/Util/WorkspaceCompare.hs

```diff
@@ -109,8 +102,6 @@ getSortByTag = mkWsSort getWsCompareByTag
 --   sorted by tag.
 getSortByXineramaRule :: X WorkspaceSort
 getSortByXineramaRule = mkWsSort getXineramaWsCompare
-
--- | Like 'getSortByXineramaRule', but uses physical locations for screens.
-getSortByXineramaPhysicalRule :: X WorkspaceSort
-getSortByXineramaPhysicalRule = mkWsSort getXineramaPhysicalWsCompare
-
+-- | Like 'getSortByXineramaRule', but allow you to use physical locations for screens. getSortByXineramaPhysicalRule :: ScreenComparator -> X WorkspaceSort
```

Comment:
Why does this comment include the type signature at the end?

Reply:
Probably a vim typo (`J` instead of `j`)